lib/node_modules/@stdlib/math/base/special/uimuldw/docs/repl.txt

```diff
@@ -0,0 +1,30 @@
+
+{{alias}}( [out,] a, b )
+    Multiplies two unsigned 32-bit integers and returns an array
```

Comment:
This description needs to be re-aligned with 80 char width. I believe `of two` can be moved to this line, which may affect subsequent lines.